### PR TITLE
nabuk regal hd

### PR DESCRIPTION
**Pull request by pazos**
https://github.com/koreader/koreader/issues/4551

wip: waiting for confirmation.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/android-luajit-launcher/333)

### CODE REVIEW

File: assets/android.lua [hunk 1]
```diff
@@ -2236,7 +2236,7 @@ local function run(android_app_state)
             delay_fast = 0
             fast, partial = wf_du + mode_partial, wf_gc16 + mode_partial
             if platform == "freescale" then
-                local wf_regal = 7
+                local wf_regal = android.prop.product == "nabukreg_hd" and 6 or 7
```
Comment (Frenzie) [inline assets/android.lua:2239]: Strange.

**Review by Frenzie — APPROVED** [review]
lgtm

File: assets/android.lua (hunk 1 above)
Comment (NiLuJe) [reply assets/android.lua:2239]: Did we ever get a look at the decompiled code of the system service that handles the refreshes on these devices?

Comment (pazos) [reply assets/android.lua:2239]: Yup, @char11 did all the work. It is documented in https://github.com/koreader/android-luajit-launcher/blob/master/app/src/main/java/org/koreader/launcher/device/epd/freescale/NTXEPDController.kt.

It is the controller used for all freescale/ntx.

IIRC we don't interface directly with the library that handles the update. the view.View class contains methods that perform native calls via jni.

Some freescale boards have system drivers that don't work with native activities and thus require that we handle all updates. On the rest we handle only full eink updates.

**Pull request merged**